### PR TITLE
Don't build for all pull requests, only build for releases

### PR DESCRIPTION
**Pull request by Graham42**
If you have a bunch of pull requests you want to merge in, the `min` files are going to constantly conflict. I would suggest we don't build for all pull requests, and instead, build for each release.

### CODE REVIEW

**Comment by Graham42** [comment]
Converted this to a pull request

**Comment by Graham42** [comment]
So I've automated the release process, just need to add documentation now

**Comment by Graham42** [comment]
Created a wiki page https://github.com/foretagsplatsen/numbro/wiki/Publishing, let me know if you have any questions

**Comment by BenjaminVanRyseghem** [comment]
I created a ticket #21 to discuss about it :smile:

File: Gruntfile.js [hunk 1]
```diff
@@ -1,45 +1,73 @@
...
+                    return {
+                        src: [path.join('languages/', fileName)],
+                        dest: path.join('dist/min/languages/', lang + '.min.js'),
+                    };
+                }))
             },
             options: {
-                preserveComments: 'some'
-            }
+                preserveComments: 'some',
```
Comment (BenjaminVanRyseghem) [inline Gruntfile.js]: extra comma ;)

File: Gruntfile.js [hunk 2]
```diff
@@ -1,45 +1,73 @@
...
+            },
```
Comment (BenjaminVanRyseghem) [inline Gruntfile.js]: here too

I am not sure this is necessary, or?

File: Gruntfile.js [hunk 3]
```diff
@@ -1,45 +1,73 @@
...
         },
-        concat: {
-            languages: {
-                src: [
-                    'languages/**/*.js'
+        bump: {
```
Comment (BenjaminVanRyseghem) [inline Gruntfile.js]: what the purpose of `bump`?

File: Gruntfile.js [hunk 4]
```diff
@@ -1,45 +1,73 @@
...
+                push: false,
+            },
+        },
+        confirm: {
+            release: {
+                options: {
+                    question: 'Are you sure you want to publish a new release' +
+                        ' with version <%= pkg.version %>? (yes/no)',
+                    continue: function(answer) {
+                        return answer.toLowerCase() === 'yes';
```
Comment (BenjaminVanRyseghem) [inline Gruntfile.js]: could we have `answer.toLowerCase() === 'yes' || answer.toLowerCase() === 'y'`

**Comment by BenjaminVanRyseghem** [comment]
It looks very good to me :+1:

**Comment by Graham42** [comment]
The reason for the extra commas in the config is to make change sets more diff-able. It's one of the failings of JSON in my mind, if you want to add an item after the last one in an object or array, you have to add a comma to the previous line. This means you will get merge conflicts if someone else also added item.

**Comment by BenjaminVanRyseghem** [comment]
kinda make sense even if I find it ugly :smiley_cat:

**Comment by uniphil** [comment]
They also break IE8, but then `uglifyjs` (and probably any minfier) will
strip them out.

**Comment by Graham42** [comment]
I only put them in the Grunt file, which is always being run by node

File: Gruntfile.js (hunk 3 above)
Comment (Graham42) [inline Gruntfile.js]: I've added this info to the wiki page

**Comment by Graham42** [comment]
So I've outline the current process in the [Publishing](https://github.com/foretagsplatsen/numbro/wiki/Publishing) wiki page. I could work on the "prompt for changelog" thing in this pull request, but I would suggest separating that out and living with this for now.

Anything else you think I should add to wrap this up?

**Comment by BenjaminVanRyseghem** [comment]
Looks really good to me, but I am not that much of a grunt guru :smile: 

So let;s try and fix things when we bump :smile:

**Pull request merged**